dasharo/releases.yaml

```diff
@@ -0,0 +1,10 @@
+releases:
+        - version: v0.1.0
+          added:
```

Comment:
@miczyg1 Can you please fill in release notes for the planned release?

Reply:
Done